Firmata.cpp

```diff
@@ -66,7 +80,8 @@ FirmataClass::FirmataClass()
   firmwareVersionCount = 0;
   firmwareVersionVector = 0;
   blinkVersionDisabled = false;
-  systemReset();
```

Comment:
I see I had overlooked that. It appears that everything is properly initialized.